src/implementations/twig/components/form-group/README.md

```diff
@@ -8,12 +8,12 @@ npm install --save @ecl/twig-component-form-group
 
 ### Parameters:
 
-- **"label"** (string) (default: '') - Label for the form group
```

Comment:
was this removal intentional?

Reply:
yes, the label is listed in the blocks; that was the case already in the twig file, so I just applied it in the readme